lua/lsp-file-operations/did-create.lua

```diff
@@ -4,7 +4,7 @@ local log = require("lsp-file-operations.log")
 local M = {}
 
 M.callback = function(data)
-  for _, client in pairs(vim.lsp.get_active_clients()) do
+  for _, client in pairs(vim.lsp.get_clients()) do
```

Comment:
if we want to keep backwards compatibility with older Neovim versions we could do
```suggestion
  local get_clients = vim.lsp.get_clients or vim.lsp.get_active_clients
  for _, client in pairs(get_clients()) do
```